controllers/translateController.js

```diff
@@ -53,11 +54,14 @@ const translateController = async ({
     cliArgEngine: options.engine,
   });
 
+  const config = await getConfig();
+
   for await (const language of languages) {
     const filesToEdit = await validateAndPromptUserJSONFiles(
       basePath,
       language.files,
       nameOfTranslation,
+      config,
```

Comment:
Btw this function only checks what files we should edit, we don't need the config here yet

```suggestion
```